Update ReadMes, mainly top-level (#58)
opendihu/optimization: BayesOpt_examples/test_functions/README.md

python BayesOpt_test_functions.py matern 0.5 const fixed_noise es stopping_xy 1
```

## Results

### The optimal Bayesian Optimization model over all test functions

> This was discussed in the mentioned [Bachelor Thesis](https://elib.uni-stuttgart.de/handle/11682/16797). The word "optimal" is very subjective. In this case we are looking for a model that balances high accuracy with a low number of evaluations. Comparing all optional models averaged over all test functions, we find the following model as the best one:

Reviewer: ```suggestion
This was discussed in the mentioned [Bachelor Thesis](https://elib.uni-stuttgart.de/handle/11682/16797). Firstly, we should clarify what we mean by "optimal". In this case we are looking for a model that balances high accuracy with a low number of evaluations. Comparing all optional models averaged over all test functions, we find the following model as the best one:
```

Do we compare "all" optional models? Definitely not. Maybe all the options in BoTorch?
